storefront/app/helpers/spree/products_helper.rb

```diff
@@ -213,5 +213,9 @@ def option_type_colors_preview_styles(option_type)
 
       Spree::ColorsPreviewStylesPresenter.new(option_type.option_values.map { |ov| { name: ov.name, filter_name: ov.name } }).to_s
     end
+
+    def default_product_properties
+      product.product_properties.joins(:property).merge(Spree::Property.available_on_front_end).sort_by_property_position
```

Comment:
```suggestion
    def product_properties(product)
      product.product_properties.joins(:property).merge(Spree::Property.available_on_front_end).sort_by_property_position
```

You need to pass `product` as this isn't a global variable